File: NoteiOS-master/代码块/BMShareView/BMShareView/ViewController.m
//
//  ViewController.m
//  BMShareView
//
//  Created by ___liangdahong on 2017/3/9.
//  Copyright © 2017年 http://idhong.com/. All rights reserved.
//

#import "ViewController.h"
#import "BMShareManager.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
}

- (void)touchesBegan:(NSSet<UITouch *> *)touches withEvent:(UIEvent *)event {

    [BMShareManager showShareManager];
}

@end

File: NoteiOS-master/swift-2.0/Swift/自定义log/CustomLog/CustomLog/ViewController.swift
//
//  ViewController.swift
//  CustomLog
//
//  Created by __liangdahong on 2016/11/1.
//  Copyright © 2016年 http://idhong.com/. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        IHLog("\(self)")
        IHLog("c", file: "a")
        
        let f1 = 1.3213
        print("\(f1)")
        print(String(format: "%.2f  f1 = \(f1)", 10.3))
        print("\(f1.bm_format("%.2f"))")
    }
}

